src/app/features/genes/components/gene-details/gene-details.component.ts

```diff
@@ -135,40 +135,45 @@ export class GeneDetailsComponent implements OnInit, AfterViewInit {
       if (params.get('id')) {
         this.geneService
           .getGene(params.get('id') as string)
-          .subscribe((gene: Gene) => {
-            this.gene = gene;
-
-            this.panels.forEach((p: Panel) => {
-              if (p.name == 'nominations' && !this.gene?.total_nominations) {
-                p.disabled = true;
-              } else if (
-                p.name == 'experimental-validation' &&
-                !this.gene?.experimental_validation?.length
-              ) {
-                p.disabled = true;
-              } else {
-                p.disabled = false;
+          .subscribe((gene) => {
```

Comment:
Could the type still be specified here?

```suggestion
          .subscribe((gene: Gene | null) => {
```

Reply:
Great question! In general I'm partial to letting TypeScript inference infer for cases like this since I feel it is obvious mostly because I prefer less verbose code.  That said, I do like to hear why others may choose to type certain variables so let me know if you have strong opinions since code style ultimately helps everyone read each other's code.  I like the explanation in this article: https://dev.to/nectres/dont-type-in-typescript-at-least-most-of-the-time-5b0j#:~:text=You%20don't%20have%20to,the%20benefits%20of%20using%20it.

Reply:
Makes sense! Thanks for sharing the article